### ISSUE TITLE
Allow for specifying xpath locator strategy when using matching xpath keywords

### ISSUE THREAD

**Issue by emanlove** [robotframework/SeleniumLibrary#526]
We shouldn't penalize users for having the leading xpath locator strategy "xpath=" in the locator that is passed to the keywords `Get Matching Xpath Count` and `Xpath Should Match X Times`. At my company, for example, it is general practice to create variables for the locators specifying the strategy. Thus when using the matching xpath count keywords we have to make a separate variable. Instead we can simply accept either and give a meaningful error otherwise; something like

``` python
if locator.startswith('//')
    locator = 'xpath=' + locator

if locator.startswith('xpath=')
    # use locator
else
    # raise an menaingful error ... "locator should be xpath and start with xpath= or //"...
```

**Comment by aaltat** [comment]
I had a similar feeling not so long time ago. It is a good idea.

**Comment by aaltat** [comment]
I am happy with the #527, specially with the latest changes and from my point of view it can be merged.

**Comment by aaltat** [comment]
I did not remember that we did have this issue but this problem has been fixed in #972
